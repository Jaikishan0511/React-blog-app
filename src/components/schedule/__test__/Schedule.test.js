import React from "react";
import renderer from "react-test-renderer";

import Schedule from "../index";
describe("Schedule", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Schedule />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
