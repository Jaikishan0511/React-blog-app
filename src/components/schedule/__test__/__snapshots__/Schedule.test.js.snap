// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule snapshot renders 1`] = `
<div>
  <ul>
    <li>
      6/5 @ Evergreens
    </li>
    <li>
      6/8 vs Kickers
    </li>
    <li>
      6/14 @ United
    </li>
  </ul>
</div>
`;
